run-tests.py

```diff
@@ -440,13 +262,6 @@ def export_notebook(ipath, opath):
     if args.doctest:
         has_run = True
         run_doc_tests()
-    # Notebook tests
-    elif args.examples:
-        has_run = True
-        run_notebooks(interpreter)
-    if args.debook:
-        has_run = True
-        export_notebook(*args.debook)
```

Comment:
Should we keep this and run `pytest` within a `subprocess`? Would keep the testing suite a bit uniform??

Reply:
We could use a subprocess, but it would come with the hassle of capturing stdout and stderr according to `pytest`'s logging functionality. I am not sure how to do it because we noted that in case a worker keeps running a notebook cell that takes a bit longer to execute, it tends to duplicate the progress bar for that notebook in its output stream. Considering we would migrate to `pytest` and eventually use it directly instead of invoking tests through `run-tests.py`, we could deprecate this command-line option instead and/or perhaps add a comment here about how the new notebook tests can be run?

Reply:
Deprecating sounds good. We should add a `ValueError` here.

Reply:
Done